Nos/Views/Profile/ProfileView.swift

```diff
@@ -88,6 +88,16 @@ struct ProfileView: View {
                             Text(.localizable.noEventsOnProfile)
                                 .padding()
                                 .readabilityPadding()
+                            
+                            SecondaryActionButton(
+                                title: .localizable.tapToRefresh
+                            ) {
+                                NotificationCenter.default.post(
+                                    name: .refresh,
+                                    object: nil,
+                                    userInfo: ["tab": AppDestination.profile]
```

Comment:
This `userInfo` isn't actually working to isolate this action to the currently displayed ProfileView. If you tap this refresh button on one ProfileView it triggers for all live ProfileViews.

Using NSNotifications to communicate between views is not a good pattern in general I think, especially because SwiftUI views don't have a stable identifier you can pass in for the `object` (unlike UIViews, where you can pass the address). I think we should refactor this so that PagedNoteListView passes emptyPlaceholder a refreshAction closure that it can call when the refresh button is tapped. So instead of `emptyPlaceholder` being a `() -> Void` closure it could be `(() -> Void) -> Void`. Then SecondaryActionButton would look like:
```
SecondaryActionButton(title: .localizable.tapToRefresh, action: refresh)
```